types/enzyme/package.json

```diff
@@ -6,8 +6,8 @@
         "https://github.com/airbnb/enzyme"
     ],
     "dependencies": {
-        "@types/cheerio": "*",
-        "@types/react": "^16"
+        "@types/react": "^16",
+        "cheerio": "^1.0.0"
```

Comment:
the next release of enzyme actually will pin this to `=1.0.0-rc.3`, since anything after that breaks, so i'm not sure this is the right approach here.